dolphin/processor/recipe.py

```diff
@@ -381,6 +388,11 @@ def get_galaxy_galaxy_recipe(self, kwargs_data_joint, epochs=2):
                     self.unfix_params('lens'),
                     self.fix_params('lens', external_shear_model_index)
                 ]
+                # fix the gamma parameter for fixed-slope recipe
```

Comment:
This is redundant here, as the same thing is done in lines 399-404. I think, if your desired behavior is to keep `gamma = 2` fixed, it should already be the case in the default `'galaxy-galaxy'` recipe. Also, when necessary, please use the variable `pl_model_index` instead of hard-coding for the index `0`.

Reply:
I think what happened is that  lines 399-404 didnt work for my version fo dolphin as the `_get_power_law_model_index` function only search for  'SPEMD' and 'SPEP'. Thus, I will remove my redundant part and add 'PEMD' support